query/src/main/scala/filodb/query/ProtoConverters.scala

```diff
@@ -180,13 +180,28 @@ object ProtoConverters {
 
   implicit class PlannerParamsToProtoConverter(pp: PlannerParams) {
     def toProto: GrpcMultiPartitionQueryService.PlannerParams = {
+      val enforcedQuotaBuilder = GrpcMultiPartitionQueryService.IndividualQuota.newBuilder()
```

Comment:
there is one defined:
Line 243.
implicit class IndividualQuotaFromProtoConverter(giq: GrpcMultiPartitionQueryService.IndividualQuota) {

Reply:
It just converts the proto to case class (``fromProto``), we need a ``toProto`` to convert the case class to gRPC object. Anywhere we deal with converting to and from proto we should use this implicit conversions rather than copying the object building logic as we have in this PR

Reply:
got you, will add toProto (only fromProto is defined now)